python/neuroglancer/pipeline/task_queue.py

```diff
@@ -183,21 +190,43 @@ def cloud_insertion(api):
         else:
             cloud_insertion(self._api)
 
+        return self
+
+    def wait(self):
```

Comment:
There were some places where it looked nicer to say `storage.put_file( ......... ).wait()`